app/models/cdr_export.rb

```diff
@@ -37,11 +37,17 @@ class FiltersModel < JsonAttributeModel
     attribute :customer_auth_external_id_eq, :integer
     attribute :failed_resource_type_id_eq, :integer
     attribute :src_prefix_in_contains, :string
+    attribute :src_prefix_in_eq, :string
     attribute :dst_prefix_in_contains, :string
+    attribute :dst_prefix_in_eq, :string
     attribute :src_prefix_routing_contains, :string
+    attribute :src_prefix_routing_eq, :string
     attribute :dst_prefix_routing_contains, :string
+    attribute :dst_prefix_routing_eq, :string
     attribute :src_prefix_out_contains, :string
+    attribute :src_prefix_out_eq, :string
     attribute :dst_prefix_out_contains, :string
+    attribute :dst_prefix_out_eq, :string
```

Comment:
please add all new filters to `spec/acceptance/rest/admin/api/cdr/cdr_exports_spec.rb` too

it's test that generates a documentation